workflow_miniscope/__init__.py

```diff
@@ -1,5 +1,5 @@
 __author__ = "DataJoint NEURO"
-__date__ = "December 15, 2020"
+__date__ = "May 1, 2021"
 __version__ = "0.0.1"
 
 __all__ = ['__author__', '__version__', '__date__']
```

Comment:
Let's put this information into a [`version.py`](https://github.com/ttngu207/workflow-array-ephys/blob/main/workflow_array_ephys/version.py) file, and modify the [`setup.py`](https://github.com/ttngu207/workflow-array-ephys/blob/main/setup.py) accordingly as well

Reply:
Good idea.  Updated both the `workflow` and `element` according to your example.